.github/workflows/clang-tidy-format.yml

```diff
@@ -13,23 +13,58 @@ concurrency:
 jobs:
   cpp-linter:
     runs-on: ubuntu-latest
+    container:
+      image: krisitanpn/teststar:cpp-linter
+      options: --user chipStarUser
     steps:
       - uses: actions/checkout@v2
-      - name: Install clang
-        run: sudo apt install clang-11 --install-suggests
+        with:
+          submodules: recursive
+      - name: Load modules
+        run: ml StdEnv; . /home/chipStarUser/venv/bin/activate;
+          echo "CPATH=$CPATH" >> $GITHUB_ENV;
+          echo "LD_LIBRARY_PATH=$LD_LIBRARY_PATH" >> $GITHUB_ENV;
+          echo "LIBRARY_PATH=$LIBRARY_PATH" >> $GITHUB_ENV;
+          echo "PATH=$PATH" >> $GITHUB_ENV;
+        shell: bash
+
+      - name: Add includes
+        run: touch `pwd`/include/chipStarConfig.hh;
+          export CPATH=`pwd`/include/:$CPATH;
+          export CPATH=`pwd`/HIP/include/:$CPATH;
+          export CPATH=`pwd`/spirv-extractor/SPIRV-Tools/include/:$CPATH;
+          echo "CPATH=$CPATH" >> $GITHUB_ENV;
+        shell: bash
+
+      - name: show env
+        run: echo "$CPATH";
+          echo "$LD_LIBRARY_PATH";
+          echo "$LIBRARY_PATH";
+          echo "$PATH";
+        shell: bash
+
       - name: C/C++ Linter
```

Comment:
I think it would be better to combine these into a single step. with 3 separate steps, you have export things to `GITHUB_ENV` whereas we can just module load prior to running the linter command.

Reply:
This is the leftover from when I intended to use the cpp-linter action. Definitely makes sense to load modules just prior to running.